### ISSUE TITLE
Notify users when the content on long links changes

### ISSUE THREAD

**Issue by lisatjide** [opengovsg/GoGovSG#18]
**Is your feature request related to a problem? Please describe.**
How do I know if the content of my long link has changed, or become not relevant? Should go.gov tell me that I should be turning the link off?

**Describe the solution you'd like**
One possible solution could be to change the colour of that particular long link on the dashboard.
And when use hovers, they should see a tooltip showing that the content has changed.

**Comment by liangyuanruo** [comment]
I'm not sure how feasible this feature is. It would require periodic scraping of the target sites in question. Additionally, the definition of the content change is unclear.

**Comment by leonardloo** [comment]
Agreed this is technically challenging from those two fronts. While periodic scraping is feasible but adds tech complexity, detecting content change can flag a lot of false positives. Leaving this issue open for discussion especially from devs in the community who have implemented anti-defacement tools before
